build_ng2.xml

```diff
@@ -117,6 +117,23 @@
       </sequential>
     </for>
   </target>
+  
+  <target name="cleanupPlugins">
+    <!-- <delete><fileset .../>...</delete> is extremely slow -->
+    <for param="path">
+      <path>
+        <dirset dir="${capstone}" defaultexcludes="false">
+          <include name="**/web/**/*.js"/>
+          <include name="**/web/**/*.js.map"/>
```

Comment:
also delete *.js.map.gz
because the plugin may have shipped a compressed map, but we dont want either.